osquery/core/watcher.h

```diff
@@ -113,12 +117,13 @@ class Watcher : private boost::noncopyable {
   static void unlock() { instance().lock_.unlock(); }
 
   /// Accessor for autoloadable extension paths.
-  static const std::map<std::string, pid_t>& extensions() {
+  static const std::map<std::string, std::shared_ptr<PlatformProcess>>&
```

Comment:
This should be post `make format`, so yes, `clang-format` appears to have not done well here...